src/cryojax/simulator/_transfer_theory/contrast_transfer_theory.py

```diff
@@ -1,27 +1,37 @@
-"""
-Models of instrument optics.
-"""
-
 from abc import abstractmethod
-from typing import ClassVar, Optional
+from typing import Optional
 from typing_extensions import override
 
 import jax.numpy as jnp
-from equinox import AbstractClassVar, AbstractVar, field, Module
+from equinox import field
 from jaxtyping import Array, Complex, Float
 
-from .._errors import error_if_negative, error_if_not_fractional, error_if_not_positive
-from ..constants import convert_keV_to_angstroms
-from ..coordinates import cartesian_to_polar
-from ..image.operators import (
-    AbstractFourierOperator,
+from ..._errors import error_if_negative, error_if_not_fractional, error_if_not_positive
+from ...constants import convert_keV_to_angstroms
+from ...image.operators import (
     Constant,
     FourierOperatorLike,
 )
-from ._config import ImageConfig
+from .._instrument_config import InstrumentConfig
+from .base_transfer_theory import AbstractTransferFunction, AbstractTransferTheory
+from .common_functions import compute_aberrated_phase_shifts
+
+
+class AbstractCTF(AbstractTransferFunction, strict=True):
+    """An abstract base class for a transfer function."""
+
+    @abstractmethod
+    def __call__(
+        self,
+        frequency_grid_in_angstroms: Float[Array, "y_dim x_dim 2"],
+        *,
+        wavelength_in_angstroms: Optional[Float[Array, ""] | float] = None,
+        defocus_offset: Float[Array, ""] | float = 0.0,
+    ) -> Float[Array, "y_dim x_dim"] | Complex[Array, "y_dim x_dim"]:
+        raise NotImplementedError
 
 
-class CTF(AbstractFourierOperator, strict=True):
+class AberratedCTF(AbstractCTF, strict=True):
```

Comment:
Is this the right name?

Perhaps sine CTF with constant amplitude contrast is the most descriptive and accurate.

Reply:
Yes wasn’t totally sure about this… definitely open to changes here.